shortfin/pyproject.toml

```diff
@@ -8,6 +8,32 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "shortfin"
+authors = [
+  {name = "SHARK Authors"},
+]
+description = "SHARK inference library and serving engine"
+readme = "README.md"
```

Comment:
Can you also update the README text in this PR to match this description?

https://github.com/nod-ai/SHARK-Platform/blob/ce6ccf88e8f106a6a88b1946076139c011a8f5e8/shortfin/README.md?plain=1#L1

(It would be weird for a project on PyPI to have "C++ library" right at the top of the page)

Reply:
Good point! Sure, I will add it to this PR prior to merging.